contracts/modules/royalty/policies/LRP/RoyaltyPolicyLRP.sol

```diff
@@ -108,8 +115,7 @@ contract RoyaltyPolicyLRP is
         uint32 licensePercent,
         bytes calldata
     ) external onlyRoyaltyModule nonReentrant {
-        IRoyaltyModule royaltyModule = ROYALTY_MODULE;
-        if (royaltyModule.globalRoyaltyStack(ipId) + licensePercent > royaltyModule.maxPercent())
+        if (ROYALTY_POLICY_LAP.getPolicyRoyaltyStack(ipId) + licensePercent > ROYALTY_MODULE.maxPercent())
             revert Errors.RoyaltyPolicyLRP__AboveMaxPercent();
```

Comment:
It would be better to add tests to cover the logic changes, such as the `onLicenseMinting()` revert in LRP due to the royalty stack in LAP being too high.

Reply:
Yes, normally would have done it but iirc cross-royalty policy graphs - which are necessary for this test - are not possible to create at the moment due to licensing side restrictions, right?

Reply:
I see.